src/map/pc.hpp

```diff
@@ -706,7 +706,11 @@ class map_session_data {
 	int united_soul[MAX_UNITED_SOULS]; // Stores the account ID's of character's who's soul is united.
 	int servant_sign[MAX_SERVANT_SIGN]; // Stores the account ID's of character's with a servant sign.
 
-	int trade_partner;
+	struct{
+		int id;
+		int lv;
```

Comment:
```suggestion
		unsigned int lv;
```
These types can be found in `mmo_charstatus`.